.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,174 @@
+name: Publish release
+
+on:
+  release:
+    types: [created]
+
+env:
+  IS_TEST: false
+  DOCKER_ORG_NAME: sanicframework
+  DOCKER_IMAGE_NAME: sanic
+  DOCKER_BASE_IMAGE_NAME: sanic-build
+  DOCKER_IMAGE_DOCKERFILE: ./docker/Dockerfile
+  DOCKER_BASE_IMAGE_DOCKERFILE: ./docker/Dockerfile-base
+
+jobs:
+  generate_info:
+    name: Generate info
+    runs-on: ubuntu-latest
+    outputs:
+      docker-tags: ${{ steps.generate_docker_info.outputs.tags }}
+      pypi-version: ${{ steps.parse_version_tag.outputs.pypi-version }}
+    steps:
+      - name: Parse version tag
+        id: parse_version_tag
+        env:
+          TAG_NAME: ${{ github.event.release.tag_name }}
+        run: |
+          tag_name="${{ env.TAG_NAME }}"
+
+          if [[ ! "${tag_name}" =~ ^v([0-9]{2})\.([0-9]{1,2})\.([0-9]+)$ ]]; then
+            echo "::error::Tag name must be in the format vYY.MM.MICRO"
+            exit 1
+          fi
+
+          year_output="year=${BASH_REMATCH[1]}"
+          month_output="month=${BASH_REMATCH[2]}"
+          pypi_output="pypi-version=${tag_name#v}"
+
+          echo "${year_output}"
+          echo "${month_output}"
+          echo "${pypi_output}"
+
+          echo "${year_output}" >> $GITHUB_OUTPUT
+          echo "${month_output}" >> $GITHUB_OUTPUT
+          echo "${pypi_output}" >> $GITHUB_OUTPUT
+
+      - name: Get latest release
+        id: get_latest_release
+        run: |
+          latest_tag=$(
+            curl -L \
+              -H "Accept: application/vnd.github+json" \
+              -H "Authorization: Bearer ${{ github.token }}" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              https://api.github.com/repos/${{ github.repository }}/releases/latest \
+              | jq -r '.tag_name'
+          )
+          echo "latest_tag=$latest_tag" >> $GITHUB_OUTPUT
```

Comment:
Using the official API action might make this easier and more readable. And I also think it would be fine to use CURL, lol.

https://github.com/octokit/request-action

Reply:
I've looked at this, and I think I'm gonna go for cURL for the time being. Main reason being because the output the action returns is in string format, and then you have to manually convert it to JSON by calling the `toJSON` function whenever you want to use it, which isn't too nice imo. Here's the diff if ur curious :D

![imagen](https://github.com/sanic-org/sanic/assets/25409753/220ce140-c634-4dbd-86f7-c14571ba17b7)